doc/sphinx/symlinks.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+from __future__ import print_function
```

Comment:
This looks ok to me, but you probably want @Kyle-B to review it (my python skills aren't very good.)